filters/filter_3_Spyware/template.txt

```diff
@@ -27,6 +27,12 @@
 !-------------------------------------------------------------------------------!
 @include "https://raw.githubusercontent.com/AdguardTeam/AdguardFilters/master/SpywareFilter/sections/cname_allowlist.txt"
 !-------------------------------------------------------------------------------!
+!-------------- CNAME mail trackers --------------------------------------------!
+!-------------------------------------------------------------------------------!
+!#if (adguard_app_ios)
```

Comment:
We should exclude iOS due to problems with Safari for iOS - the real limit is much lower than 150k rules, and the content blocker size is limited to 10 MB. CC @AdguardTeam/filters-maintainers 
```suggestion
!#if (!adguard_app_ios)
```

Reply:
Maybe then add safari content blocker as well?

Reply:
it's iOS-specific issue, AdGuard for Safari does not have that content blockers size limit